### DISCUSSION TITLE
HA OneDrive Add-on Not Starting

### DISCUSSION THREAD

**Discussion by Briandr73** [lavinir/hassio-onedrive-backup#198] [General]
Hi @lavinir 

The OneDrive add-on was able to do a backup\synch and then it stopped. I tried to start it and just got a spinning circle. I see nothing in the logs except for where it indicated the successful backup\synch. Also, when I click on 'Configuration' I am getting a message saying this Add-on has no configuration. Wondering what I am supposed to be doing, but not doing. It did create a HA backup as noted above and it synch'd across to OneDrive, but it appears I am missing something.

Update:

Full restart of HA allowed the add-on to start, but I am seeing nothing under Configuration except message saying Add-on has no configuration.

**Comment by lavinir** [comment]
All configuration has moved into the addon Web Interface a while back. You have everything there and all configuration is under the "Settings" tab

**Reply by Briandr73** [reply]
Thank you for pointing that out. I guess I might have stumbled across that eventually. I am noticing my backups are only 58mb (today) and 78mb (yesterday). When I was running backups within HA they were usually more than 200mb. I ran the below command from within HA:

 action:
  - service: hassio.backup_full
    data:
      compressed: false

Could it be your add-on is compressing the backup? That is my guess because command I am running is not compressing the backup. Also, does your backup also take care of HACS? 

Many thanks because I did not want to use Google and Dropbox was problematic.

**Reply by lavinir** [reply]
Yes the backups are compressed. 
The backups themselves use Home Assistant to backup so it's a standard Hass backup. Anything inside the config folder (includes HACS) is also part of those backups.
You can always take a look at them in the Backups menu in Home Assistant, as while they are local you can view them there as any other backup.

**Comment by judaction** [comment]
Hi, my onedrive backup addon is not starting after having it deinstalled and re-installed including the authentication part for my personal microsoft onedrive. Nothing shown under COnfioguration and as well Settings. The log says (replaced auth. code with *****):

 [1] Info: No new settings file. Attempting to restore from legacy configuration
05/09/2024 14:52:18 [4] Info: Anonymous Telemetry Disabled
05/09/2024 14:52:18 [4] Info: Backup interval configured to every 24 hours
05/09/2024 14:52:18 [4] Warning: Token cache is empty
05/09/2024 14:52:21 [10] Info: To sign in, use a web browser to open the page https://microsoft.com/devicelogin and enter the code ******* to authenticate.
05/09/2024 14:53:40 [11] Info: Found 1 backups to upload.
05/09/2024 14:53:40 [11] Info: Uploading hass_backup_2024-05-09-14-26_Man (05/09/2024 14:26:19)
05/09/2024 14:53:40 [11] Info: Fetching Local Backup (Slug:802b132f)
05/09/2024 14:53:48 [16] Info: Backup (802b132f) fetched successfully
05/09/2024 14:53:49 [6] Info: Starting file upload. (Size:591851520 bytes. Attempt: 1/3)
05/09/2024 15:08:59 [26] Info: Upload completed successfully
05/09/2024 15:09:00 [21] Info: Reached Max Online Backups (10)
05/09/2024 15:09:00 [20] Info: Found 1 backups to delete from OneDrive.
05/09/2024 15:09:00 [20] Info: Deleting item: hass_backup_2024-04-23-13-33.tar
05/09/2024 15:09:00 [20] Info: Reached Max Local Backups (10)
05/09/2024 15:09:00 [20] Info: Removing 1 local backups

The addon still shows red icon. I have restarted HA several times now.
Any help would be appreciated.

**Reply by lavinir** [reply]
I don't understand, logs are showing addon is running but you aren't able to open the UI ?
Please open an issue if relevant.

**Reply by judaction** [reply]
Sorry, it works now, I had a serious issue with my WLAN but now attached the home assistant via cable to the network.
